src/main/java/de/doubleslash/keeptime/App.java

```diff
@@ -239,7 +243,7 @@ private void initialisePopupUI(final Stage primaryStage) throws IOException {
       final ViewControllerPopup viewControllerPopupController = loader.getController();
       viewControllerPopupController.setStage(popupViewStage);
 
-      if (!OS.isLinux()) {
+      if (OS.isWindows()) {
```

Comment:
FileOpenHelper and BrowserHelper most likely do not work for MAC due to if/else should resovle to not supported:
https://github.com/doubleSlashde/KeepTime/blob/019bffd40628a53886a4ea2195f1ca71af7b9e52/src/main/java/de/doubleslash/keeptime/common/BrowserHelper.java#L38